tests/tensorflow/nightly/vision-coco.libsonnet

```diff
@@ -42,9 +47,15 @@ local utils = import 'templates/utils.libsonnet';
     modelName: 'vision-retinanet',
     scriptConfig+: {
       experiment: 'retinanet_resnetfpn_coco',
-      paramsOverride+: {
-        task+: {
-          annotation_file: '$(COCO_DIR)/instances_val2017.json',
+    },
+  },
+  local maskrcnn = common.TfVisionTest + coco {
+    modelName: 'vision-maskrcnn',
+    scriptConfig+: {
+      experiment: 'maskrcnn_resnetfpn_coco',
+      paramsOverride: {
+        trainer: {
+          train_steps: 400,
```

Comment:
FYI, this is also setting the convergence train steps to 400, and the functional test still gets overridden to 200 steps by `functional`

Reply:
ah ok I see